Control/DeepSeq.hs

```diff
@@ -179,6 +179,13 @@ instance GNFData arity a => GNFData arity (M1 i c a) where
   grnf args = grnf args . unM1
   {-# INLINEABLE grnf #-}
 
+#if __GLASGOW_HASKELL__ >= 800
+  -- @since 1.4.5.0
+instance GNFData Zero (URec a) where
```

Comment:
This should work just as well for `NFData1`, so replace `Zero` with `arity`.

Reply:
Thanks! Will do!

Reply:
resolved